app/models/rubygem.rb

```diff
@@ -33,9 +33,11 @@ class Rubygem < ApplicationRecord
   after_create :create_gem_download
   before_destroy :mark_unresolved
 
-  MFA_RECOMMENDED_THRESHOLD = 165_000_000
+  MFA_RECOMMENDED_THRESHOLD = 150_000_000
+  MFA_REQUIRED_THRESHOLD = 165_000_000
 
   scope :mfa_recommended, -> { joins(:gem_download).where("gem_downloads.count > ?", MFA_RECOMMENDED_THRESHOLD) }
+  scope :mfa_required, -> { joins(:gem_download).where("gem_downloads.count > ?", MFA_REQUIRED_THRESHOLD) }
```

Comment:
with respect to the TODO you added in to clarify owner-defined mfa requirement vs policy mandated mfa requirement, perhaps we can rename this to `mfa_required_from_downloads`?